samples/bucketLock.js

```diff
@@ -47,12 +47,12 @@ async function getRetentionPolicy(bucketName) {
   // Creates a client
   const storage = new Storage();
   const [metadata] = await storage.bucket(bucketName).getMetadata();
-  if (metadata.hasOwnProperty('retentionPolicy')) {
+  if ('retentionPolicy' in metadata) {
```

Comment:
This isn't too conventional in JS. `metadata.hasOwnProperty('retentionPolicy')` is like the over-verbose, "I want to be 1000% sure" style, and ``retentionPolicy' in metadata` is like the cool kid, "I don't need semicolons" attitude. Just `if (metadata.retentionPolicy)` is the middle ground :)

Reply:
Thanks @stephenplusplus.

The reason why I stopped using metadata.hasOwnProprety was due to a linting issue.. that I can't recall at the moment. I'll revert.

Reply:
The relevant PR #784 to make linter happy is merged.